test/gui/shared/scripts/helpers/api/sharing_helper.py

```diff
@@ -11,6 +11,23 @@
     {"user": 0, "group": 1, "public_link": 3, "federated_cloud_share": 6}
 )
 
+PERMISSIONS={
+    "read": 1,
```

Comment:
do we need this permission?

Reply:
yes, we need to send permission in the payload

Reply:
and `"Download / View"`? both have same value

Reply:
i took this info from the roles api, since  `Download / View` have permission equivalent to `read` only